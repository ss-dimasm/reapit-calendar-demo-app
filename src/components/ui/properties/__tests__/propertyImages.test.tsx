import React from 'react';

import { render, shallow } from 'enzyme';

import PropertyImages from '../propertyImages';

describe('Property Images', () => {
  it('should match a snapshot', () => {
    const wrapper = shallow(<PropertyImages />);
    expect(wrapper).toMatchSnapshot();
  });

  it('should render static HTML', () => {
    const wrapper = render(<PropertyImages />);
    expect(wrapper.text()).toEqual('No images is available');
  });
});
