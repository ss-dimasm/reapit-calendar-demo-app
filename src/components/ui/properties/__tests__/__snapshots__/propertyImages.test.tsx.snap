// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Property Images should match a snapshot 1`] = `ShallowWrapper {}`;
